lib/commands/general.js

```diff
@@ -93,5 +93,16 @@ commands.launchApp = iosCommands.general.launchApp;
 
 commands.closeApp = iosCommands.general.closeApp;
 
+commands.keys = async function (keys) {
+  if (!this.isWebContext()) {
+    throw new errors.UnknownError('Command should be proxied to WDA');
+  }
+  let el = await this.active();
```

Comment:
interesting, so there's no "send keys to active element" atom we can use directly? I find it odd we have to get the active element ourselves.